pipeline/update_watchdog.sh

```diff
@@ -56,8 +68,13 @@ fi
 if [[ "${push}" == true ]]; then
   pushd ${nodedb}
   git reset --hard origin/master
-  git clean -xdf
-  git pull
+  if git remote -v | grep -q nodedb; then
+    git clean -xdf
+    git pull
+  else
+    echo "!! [ERROR] Not in nodedb directory. !!"
+    popd
+    exit
   popd
```

Comment:
additional popd?

Reply:
Second one only executes if everything succeeds, right?
First one executes before exit to return to the pipeline directory before exiting.